docs/database_info/biosample.rst

```diff
@@ -20,10 +20,10 @@ NCBI BioSamples are mapped into the Chado.biomaterial table.
 Undecided mappings
 ------------------
 
-We don't currently know how we will mapping analyses to biomaterials in Chado.  Assemblies that are listed in BioSample records are therefore ignored currently.
+We don't currently know how we will map analyses to biomaterials in Chado.  Assemblies that are listed in BioSample records are therefore ignored currently.
 
 
 Attributes
 ----------
 
-This module does not currently map attributes to ontology terms.  Instead, all attributes are put into a "NCBI Property" controlled vocabulary.  The full attribute set can be downloaded at `NCBI <https://www.ncbi.nlm.nih.gov/biosample/docs/attributes/?format=xml>`_
+This module does not currently map attributes to ontology terms.  Instead, all attributes are put into a "NCBI Property" controlled vocabulary. Suggested attribute - ontology term mappings for the `Plant 1.0<https://www.ncbi.nlm.nih.gov/biosample/docs/packages/Plant.1.0/>`_ and `Invertebrate 1.0<https://www.ncbi.nlm.nih.gov/biosample/docs/packages/Invertebrate.1.0/>`_ BioSample packages are available here: https://data.nal.usda.gov/dataset/data-tripal-eutils-tripal-module-increase-exchange-and-reuse-genome-assembly-metadata.  The full attribute set can be downloaded at `NCBI <https://www.ncbi.nlm.nih.gov/biosample/docs/attributes/?format=xml>`_.
```

Comment:
Links are formatting weird, I think there needs to be a space between `1.0` and `<` for the two links (Plant 1.0 and Invertebrate 1.0).